integration/storagetest/maptests.go

```diff
@@ -129,3 +135,53 @@ func (*mapTests) TestMapLayout(ctx context.Context, t *testing.T, s storage.MapS
 		t.Errorf("Layout(): %v", err)
 	}
 }
+
```

Comment:
Missing tests:
* Requesting a Tile ID which does not exist
* Requesting a Tile by using an invalid ID (e.g. the ID is not aligned at the byte boundary)
* Playing with multiple revisions; at the moment the tests will pass if the revision parameter is ignored and only the latest revision of the tiles is returned

Reply:
Added todos